docs/tutorial.rst

```diff
@@ -0,0 +1,987 @@
+.. _tutorial:
+
+A first app with Kinto
+######################
+
+There are actually two kinds of app that you may want to use Kinto with:
+
+  - App to sync user data between her devices;
+  - App to sync and share data between users, with fined-grained permissions.
+
+Most of the app will fit in one or the other.
+
+
+Sync user data between devices
+==============================
+
+Let say that we want to do a TodoMVC backend that will sync user tasks
+between her devices.
+
+
+In order to separate data between each user, we will use the default
+user bucket to create our collection:
+
+.. code-block:: http
```

Comment:
httpdomain is more to document RESTAPI's than showing example of usage of it (even if there are example in the documentation of it)

I am not sure it is relevant to use it here.